src/features/Planner/data/utils.ts

```diff
@@ -146,6 +147,7 @@ export const selectList = (state, id) => {
         state = addTask(state, id, "");
     }
     PlanApi.getDescendantsAsList(state.activeListId);
+    history.push(`/plan/${id}`);
```

Comment:
@switzerb this is the magic line. Without it, the now-deleted plan is reselected via the route match: GONK!

Reply:
wow -- i would not have predicted that. nice find!